students/WesWang/lesson8/circle.py

```diff
@@ -0,0 +1,51 @@
+import math
+
+class Circle:
+    def __init__(self, radius):
+        self.radius = radius
+    def __str__(self):
+        return "Circle with radius: " + str(self._radius)
+    def __repr__(self):
+        return f"Circle({self._radius})"
```

Comment:
in the __init__, you use self.radius, but now self._radius 
you should be consitant.